boba-chain-ops/genesis/config_test.go

```diff
@@ -24,6 +24,10 @@ func TestConfigMarshalUnmarshal(t *testing.T) {
 
 	require.NoError(t, err)
 	require.JSONEq(t, string(b), string(encoded))
+
+	err = decoded.Check()
+	require.Error(t, err)
```

Comment:
No need to require Error and ErrorContains.  ErrorContains will fail if there is no error.

Reply:
Fixed